.bit/responses/1.5-Week 1 Step 5.md

```diff
@@ -11,38 +11,10 @@ Week 1 Step 5 ⬤⬤⬤⬤⬤◯◯◯ | 🕐 Estimated completion: 25-35 minute
 
 ### ✅  Tasks:
 - [ ] Add on to the HTTP Trigger in the previous issue to check if the user’s parameter input of “password” equals “letmein”
-- [ ] If it doesn’t equal the correct password, the function should return “Access denied.” and **not** return the time.
+- [ ] If it doesn’t equal the correct password, the function should return “Access denied.”
+- [ ] To move on, place your function URL in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `HTTP_ENDPOINT` so we can check your function. Remember to commit the function's code in a file named `letMeIn.js` to the root of the `week1` branch!**
```

Comment:
This markdown doesn't work, and maybe just add a few resources to basic JS if statements etc. You'll also need to provide information on how to "return" information from the serverless function:

(Don't quote me on this syntax lol)
```
context.res = {
}
```

Reply:
the markdown works for me..? 
![image](https://user-images.githubusercontent.com/43687874/118924889-f368d280-b8fa-11eb-8c47-ca36af78d1d5.png)